tools/scripts/config_ftd2xx.py

```diff
@@ -0,0 +1,66 @@
+#!/bin/python
+
+import argparse
+import urllib.request
+import zipfile
+import tarfile
+import io
+import os
+import shutil
+
+description_help='''Download ftd2xx libraries
+Examples:\n
+    	>python config_ftd2xx.py .
+'''
+
+def parse_input():
+	parser = argparse.ArgumentParser(description=description_help,\
+				formatter_class=argparse.RawTextHelpFormatter)
+	parser.add_argument('ftd2xx_lib', help="Path to ftd2xx library location")
+	args = parser.parse_args()
+
+	return (args.ftd2xx_lib)
+
+(ftd2xx_lib) = parse_input()
+
+filename = 'LibMPSSE_1.0.7.zip'
+urllib.request.urlretrieve('https://ftdichip.com/wp-content/uploads/2025/02/LibMPSSE_1.0.7.zip', filename)
```

Comment:
yeah well this likely downloads the file in tools/scripts, i don't really like that, you should download it under `libraries/ftd2xx` and add that path to .gitignore